src/repl.zig

```diff
@@ -1904,3 +1919,40 @@ test "repl.zig: Handle parsing errors" {
         try std.testing.expectError(t.err, result);
     }
 }
+
+test "repl.zig: Parser fails for operations not supporting multiple objects" {
+    const tests = [_]struct {
+        in: []const u8,
+        err: anyerror,
```

Comment:
```suggestion
        string []const u8,
        result: anyerror,
```

Per TigerStyle, don't abbreviate.

Also per TigerStyle, where names go together in pairs like here, try to choose variable names with the same number of characters. This has the 2nd order effect of letting similar code all line up in the source. It's subtle but when you know to look for it, you'll see it all over the code base, and it's intentional.

Reply:
For example, if you were writing a routine to copy from a to b, you wouldn't use names like `src` and `dest` which are common in C code, but rather `source` and `target` (in full, not abbreviated, and they line up symmetrically, which makes it easy on the eye and easy for the reader to compare similar blocks of code at a glance).